automation/test.sh

```diff
@@ -27,6 +27,18 @@ readonly TEMPLATES_SERVER="gs://kubevirt-vm-images"
 readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
 
 
+# Skip if it's docs changes only
```

Comment:
Thanks, I did consider it, and I would liket o keep the comment, and I would like to keep it not in a function - mainly to align with the rest of the doc, and I don't see how it can be reused.